cypress/integration/buy-shirt.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  MenuContentPage,
+  ProductList,
+  Login,
+  ShippingStep,
+  AddressStep,
+  PaymentStep,
+  ShoppingCart,
+} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productList = new ProductList();
+const login = new Login();
+const shipping = new ShippingStep();
+const address = new AddressStep();
+const payment = new PaymentStep();
+const shopping = new ShoppingCart();
+
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productList.selectTShirt();
+    productList.proceedToCheckout();
+    shopping.proceedToCheckout();
+    login.typeEmail();
+    login.typePwd();
+    login.signIn();
```

Comment:
This three can be one step, but it should be reusable, you should pass by parameter the user-password